src/main/java/com/smartcar/sdk/AuthClient.java

```diff
@@ -97,13 +97,12 @@ public static String getUserId(String accessToken) throws SmartcarException {
 
     try {
       json = new Gson().fromJson(response.body().string(), JsonObject.class);
+      return json.get("id").getAsString();
     } catch (IOException ex) {
       throw new SmartcarException(ex.getMessage());
-    } catch (NullPointerException ex) {
-      throw new SmartcarException("Received an empty response body.");
+    } catch (NullPointerException ex)  {
+      throw new SmartcarException(ex.getMessage());
```

Comment:
should we just forward the null pointer exception here?

Reply:
Agreed with this, we probably shouldn't coerce an NPE into a `SmartcarException`

Reply:
Done, forgot to commit it, :upside_down_face: